steps/validate.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# Show Action Version
+echo "Using Rainforest GitHub Action v${RF_ACTION_VERSION}"
+
+# Ensure results directory is there
+mkdir -p results/rainforest
+
+echo "::add-mask::${RF_TOKEN}"
+
+# Define error helper
+error () {
+  echo "::error ::${1}"
+  echo "error=${1}" >> "$GITHUB_OUTPUT"
+  exit 1
+}
+
+# Validate token
+if [ -z "${RF_TOKEN}" ] ; then
+  error "Token not set"
+fi
+
+# Validate run_group_id
+if ! echo "${RF_RUN_GROUP_ID}" | grep -Eq '^[0-9]+$' ; then
+  error "run_group_id not a positive integer (${RF_RUN_GROUP_ID})"
+fi
+
+# Check for rerun
+if [ -s .rainforest_run_id ] ; then
+  RAINFOREST_RUN_ID=$(cat .rainforest_run_id)
+  rm .rainforest_run_id
+  echo "Rerunning Run ${RAINFOREST_RUN_ID}"
+
+  RUN_COMMAND="rerun \"${RAINFOREST_RUN_ID}\" --skip-update --token \"${RF_TOKEN}\" --junit-file results/rainforest/junit.xml --save-run-id .rainforest_run_id"
```

Comment:
Could use single quotes here and save yourself some escaping? e.g.
```
RUN_COMMAND="rerun '${RAINFOREST_RUN_ID}' --skip-update ..."
```

Reply:
I did that, then reverted because I needed to update all the tests and more crucially the escaping I added in the first commit, and I'd rather not figure out how to get the escaping working again 🙇🏼